src/Spec2-CommonWidgets-Tests/SpFilteringListPresenterTest.class.st

```diff
@@ -7,8 +7,9 @@ Class {
 	#instVars : [
 		'listWithFilter'
 	],
-	#category : 'Spec2-CommonWidgets-Tests',
-	#package : 'Spec2-CommonWidgets-Tests'
+	#category : 'Spec2-CommonWidgets-Tests-Core',
+	#package : 'Spec2-CommonWidgets-Tests',
+	#tag : 'Core'
```

Comment:
convention is actually put tag before package (because then the template will work correctly, otherwise it doesn't). I do not think this is an issue in monticello, however... maybe @jecisc knows ?

Reply:
I added the tag after the package in the Tonel format because I did not really think about the order in other places :( 

So this is not related to this PR